app/components/elements/forms/collection_radio_buttons_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form.collection_radio_buttons field_name, collection, value_method, text_method do |builder| %>
```

Comment:
I appreciate referring to form builder instances as `builder` and not `form` to keep that clear! I suspect we have not been consistent about that (I'm likely a guilty party).

Reply:
I was just following the rails code (which used "b" but I expanded to "builder").